src/webservice/Router.h

```diff
@@ -0,0 +1,109 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#pragma once
+
+#include <memory>
+#include <regex>
+#include <unordered_map>
+#include <vector>
+
+#include <proxygen/lib/http/HTTPMethod.h>
+
+#include "common/cpp/helpers.h"
+
+namespace proxygen {
+class RequestHandler;
+class HTTPMessage;
+}   // namespace proxygen
+
+namespace nebula {
+namespace web {
+
+using PathParams = std::unordered_map<std::string, std::string>;
+using ReqHandlerGenerator = std::function<proxygen::RequestHandler *(PathParams &&)>;
+
+class Route final {
```

Comment:
I wonder if it's applicable to make `Route` a subclass of `Router`.

Reply:
In my opinion, Router is a manager for all routes.